app/assets/javascripts/issue_renderer.js

```diff
@@ -1,68 +1,62 @@
 function renderIssue(issue, option) {
-  var labels = issue.labels.map(function(label) {
-    return label.name;
-  });
-  var issueType = getIssueType(issue, labels);
-  var selector = issueSelector(issueType);
+  var labelNames = pluckLabelNames(issue.labels);
+
+  var label = issueLabel(labelNames);
+  var columnId = getColumnId(issue, labelNames);
+  var body = sanitizeIssueBody(issue.body);
 
   if (option == "prepend") {
-    $(selector).prepend(issueTemplate(issue, labels));
+    $(columnId).prepend(issueTemplate(issue, body, label, issue.number, issue.title));
   } else {
-    $(selector).append(issueTemplate(issue, labels));
+    $(columnId).append(issueTemplate(issue, body, label, issue.number, issue.title));
   }
 }
 
-function getIssueType(issue, labels) {
-  if (issue.state == "closed") {
-    return "complete";
-  } else if (labels.includes("in progress")) {
-    return "in progress";
-  } else {
-    return "backlog";
-  }
+function pluckLabelNames(labels) {
+  return labels.map(function(label) { return label.name; });
 }
 
-function issueTemplate(issue, labels) {
-  var body     = issueBody(issue);
-  var label    = issueLabel(labels);
+function issueLabel(labelNames) {
+  var labelNodeData = getLabelNodeData(labelNames) // returns [Label Text, Label Style]
+  return getLabelDOMNode(labelNodeData);
+}
 
-  return "<div class='issue panel panel-warning ui-widget-content'>" +
-         "<div class='panel-heading'>" +
-         "<span class='issue-number'>" + issue.number + "</span>" + " - " + issue.title +
-         "</div>" +
-         "<div class='panel-body'>" +
-         "<span>" +
-         body +
-         "</span>" +
-         label +
-         "</div>"+
-         "</div>"
+function getLabelNodeData(labels) {
+  if (labels.includes("help wanted")) {
+    return ["Help Wanted", "label-success"];
+  } else if (labels.includes("enhancement")) {
+    return ["Enhancement", "label-info"];
+  } else if (labels.includes("bug")) {
+    return ["Bug", "label-danger"];
```

Comment:
I like how you isolated the LabelNodeData task to it's own function. I can't think of a good way to do so right now but it might be worth containing the conditional and return logic further.